src/Symfony/Component/Intl/Transliterator/EmojiTransliterator.php

```diff
@@ -72,12 +72,12 @@ public function createInverse(): self
 
         public function getErrorCode(): int|false
         {
-            return $this->transliterator?->getErrorCode() ?? 0;
+            return isset($this->transliterator) ? $this->transliterator->getErrorCode() : 0;
```

Comment:
Is this really needed? Do the tests fail without the patch? Doesn't the `??` already guard against these?

Reply:
The null-coalesce operator doesn't help here when calling a method (that's why this went unnoticed in the linked discussion). The tests are failing without these changes (see also https://3v4l.org/HBV0b).